packages/arb-ts/src/lib/networks.ts

```diff
@@ -30,6 +30,7 @@ export interface Network {
   confirmPeriodBlocks?: number
   blockTime?: number //seconds
   rpcURL: string
+  inbox: string
```

Comment:
I believe this is more appropriate in the `tokenBridge: TokenBridge` object.
Even better could be creating a new `ethBridge` object inside Network.
We can then include addresses for inbox/outbox/rollup/sequencer inbox etc

Reply:
The type would be `ethBridge: EthBridge | undefined` since there are no ethBridge contracts deployed when in a L2 network